src/layouts/split/index.tsx

```diff
@@ -1,51 +1,43 @@
-import { Box } from "@mui/material";
-import Grid from "@mui/material/Unstable_Grid2";
-import { SplitLayoutTypes } from "./types";
+import { Box, useTheme } from '@mui/material'
+import Grid from '@mui/material/Unstable_Grid2'
+import { SplitLayoutTypes } from './types'
 
-const defaultGridStyles = [
-  {
-    backgroundColor: "background.dark",
-    color: "white",
-  },
-  {
-    backgroundColor: "background.light",
-    color: "black",
-  },
-];
+const SplitLayout = ({ children, grid = [4, 8], gridStyles, columns = 12 }: SplitLayoutTypes) => {
+  const theme = useTheme()
+
+  const defaultGridStyles = [{
```

Comment:
nitpick: Export, from `./types`, `GridStyleType` and then type `defaultGridStyles:GridStyleType`, just so that it will throw an error in case someone tries to change that to a different format